src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
+
+    subtract(number) {
+      calculator.result -= number;
+    },
+
+    multiply(number) {
+      calculator.result *= number;
+    },
+
+    divide(number) {
+      calculator.result /= number;
+    },
+
+    reset() {
+      calculator.result = 0;
```

Comment:
You can use this here

When you use constructors, classes, or something like that `this` looks to service which call your method
`calculator.reset()` -> this on reset is a calculator
But `add` is a callback, the function doesn’t have this:
`add()` -> this isn’t exit here